barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_rollup_flavor.hpp

```diff
@@ -43,7 +43,12 @@ class UltraRollupFlavor : public bb::UltraFlavor {
         bool contains_ipa_claim;
         IPAClaimPubInputIndices ipa_claim_public_input_indices;
 
-        bool operator==(const VerificationKey&) const = default;
+        bool operator==(const VerificationKey& other) const
```

Comment:
I think here you can just define the `default` and it will be equivalent

Reply:
No my understanding is that defining the `default` here will invoke the element-wise default on the child class (this class) but the custom default on the parent (the one you defined)

Reply:
But in any case you should go back to using `default` for both child and parent since I remove the ocs_vkey from the VK altogether